CONTRIBUTORS.md

```diff
@@ -78,4 +78,4 @@ From oldest to newest contributor, we would like to thank:
 - [Matt Hammerly](https://github.com/mhammerly)
 - [Christian Vonrüti](https://github.com/alshain)
 - [Alessandro Vergani](https://github.com/Loghorn)
-
+- [Ethan Slattery](https://github.com/CrustyAuklet)
```

Comment:
please don't remove the blank line at the end of this file

Reply:
fixed this in rebase